projects/laji/src/app/shared/service/form.service.ts

```diff
@@ -34,14 +35,15 @@ export class FormService {
   constructor(
     private lajiApi: LajiApiService,
     private http: HttpClient,
-    private userService: UserService
+    private userService: UserService,
+    private translate: TranslateService
   ) {}
 
   static isTmpId(id: string): boolean {
     return !id || (id && id.indexOf(FormService.tmpNs + ':') === 0);
   }
 
-  getForm(formId: string, lang: string): Observable<Form.SchemaForm> {
+  getForm(formId: string, lang = this.currentLang): Observable<Form.SchemaForm> {
```

Comment:
better default value could be `this.translate.currentLang` since this.currentLang is set after this

Reply:
Good catch, thanks!